blueflood-core/src/test/java/com/rackspacecloud/blueflood/types/MaxValueTest.java

```diff
@@ -32,97 +39,375 @@ public void setUp() {
     }
 
     @Test
-    public void testMaxValueForDoubleMetrics() throws IOException {
-        for (double val : TestData.DOUBLE_SRC) {
-            max.handleFullResMetric(val);
-        }
-        Assert.assertTrue(max.isFloatingPoint());
-        Assert.assertEquals(90.48232472545334, max.toDouble(), 0);
+    public void fullResInitialDoubleSetsValue() {
+
+        // when
+        max.handleFullResMetric(123.45d);
```

Comment:
I think initializing in each test method is a good thing, right? It allows us to parallelize at test method level, if we ever want to.

But... since it was like that from the beginning, and maybe we won't ever parallelize our tests at method level, I am ok with leaving it as is.